pulp_rpm/app/viewsets/repository.py

```diff
@@ -225,10 +225,13 @@ def sync(self, request, pk):
             )
             if repository.retain_package_versions > 0:
```

Comment:
does it make sense to change this for a better readability into :
`if repository.retain_package_versions > 0 and sync_policy in (..)`

Reply:
unless we have future additional limitations with `SYNC_POLICIES.MIRROR_COMPLETE, SYNC_POLICIES.MIRROR_CONTENT_ONLY` then probably let's let it be as is

Reply:
Yeah that was (I think) the reason it's like this.